### PR TITLE
wasm-pack: update stale lockfile

### PR DESCRIPTION
**Pull request by richiksc**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

cc @carlocab [CI-force-arm]

### CODE REVIEW

File: Formula/wasm-pack.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 class WasmPack < Formula
   desc "Your favorite rust -> wasm workflow tool!"
   homepage "https://rustwasm.github.io/wasm-pack/"
+  # Remove "Cargo.lock" resource at version bump!
   url "https://github.com/rustwasm/wasm-pack/archive/v0.9.1.tar.gz"
   sha256 "56930d1f15bbcc85771e9b8e5c1b7bb6734b2c410c353ecd11eae828e35d5fb0"
```
Comment (carlocab) [inline Formula/wasm-pack.rb:6]:
```suggestion
  sha256 "56930d1f15bbcc85771e9b8e5c1b7bb6734b2c410c353ecd11eae828e35d5fb0"
  license all_of: ["Apache-2.0", "MIT"]
```

**Comment by carlocab** [comment]
That test outcome does not look right at all. Probably should be reported upstream.

**Comment by richiksc** [comment]
> That test outcome does not look right at all. Probably should be reported upstream.

Yeah, I got the same test failure locally. Let's see if `wasm-pack new works when installed from master.

**Comment by richiksc** [comment]
Interesting, this is a known issue upstream: https://github.com/rustwasm/wasm-pack/issues/907

However, we only get that error on 11-arm CI. They are not on Apple Silicon, I'm assuming, as the issue was opened in August.

**Comment by richiksc** [comment]
I also have the same issue when I build from `master` with an updated lockfile.

**Pull request closed without merging**